pkg/api/v1/status/project_team.go

```diff
@@ -0,0 +1,14 @@
+package status
+
+import "github.com/mongodb/mongodb-atlas-kubernetes/pkg/api/v1/common"
+
+type ProjectTeamStatus struct {
+	Teams  []ProjectTeamRef `json:"teams"`
+	Status bool             `json:"status"`
```

Comment:
As global status can be deduced from the condition we might want to restructure this section and make this status a list of objects with references and separate statuses for each one 💡

Reply:
I thought about this initially, but it wouldn't make sense once the assignment to the project is a batch and not an individual operation. Maybe make sense to remove this status flag from here. what do you think @fabritsius?

Reply:
Yeah, since we have a condition I think it's ok to remove it